src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_ID = 0;
+    private static final int AMOUNT_ID = 1;
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String SEPARATOR = ",";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] lines = readFromFile(fromFileName).split(System.lineSeparator());
+        String[][] data = new String[lines.length][2];
+        String[] buffer;
+        for (int i = 0; i < lines.length; i++) {
+            buffer = lines[i].split(SEPARATOR);
+            data[i][OPERATION_ID] = buffer[OPERATION_ID];
+            data[i][AMOUNT_ID] = buffer[AMOUNT_ID];
+        }
```

Comment:
remove this block of code and rewrite your logic, so you could calculate all needed values based on lines array